README.md

```diff
@@ -12,5 +12,16 @@ on the
 [public-credentials](http://lists.w3.org/Archives/Public/public-credentials/)
 mailing list as well.
 
+### Testing Examples
+
+The example Verifiable Credentials are available in the [`examples/`](examples/)
+folder and can be tested directly using various tools such as the
+[JSON-LD CLI](https://github.com/digitalbazaar/jsonld-cli).
+
+```sh
+$ npm i -g jsonld-cli
```

Comment:
Could mention the `npx jsonld-cli ...` method too.

Reply:
Meh. 😉 Primary point is to show it's usable via npm "stuff." Anyone who knows about npx and wants to use that, knows they can.